src/Paging.js

```diff
@@ -0,0 +1,68 @@
+import React, { Component } from "react";
+import './App.css';
+export default class Paging extends Component {
+    constructor() {
+        super();
+        this.state = { page: 1 };
+    }
```

Comment:
You don't have to use a constructor to set up state. You can achieve the same result by setting up state like this:
```
state = {
    page: 1
}
```